cli/cli.ts

```diff
@@ -0,0 +1,151 @@
+      description: "API type",
+      choices: ["rest", "graphql"],
+    })
+    .option("d", {
+      alias: "database",
+      description: "Database",
+      choices: ["postgresql", "mongodb"],
+    })
+    .option("au", {
+      alias: "auth",
+      type: "boolean",
+      description: "Include built-in auth features",
+    });
+
+  if (argv.backend === "typescript") {
+    argv.backend = "TypeScript (Node/Express)";
+  } else if (argv.backend === "python") {
+    argv.backend = "Python (Flask)";
+  }
+  if (argv.api === "rest") {
+    argv.api = "REST";
+  } else if (argv.api === "graphql") {
+    argv.api = "GraphQL";
+  }
+  if (argv.database === "postgresql") {
+    argv.database = "PostgreSQL";
+  } else if (argv.database === "mongodb") {
+    argv.database = "MongoDB";
+  }
+  return argv;
+};
+
+const promptOptions = async (options: Options) => {
+  const prompts = [];
+  if (!options.backend) {
+    prompts.push({
+      type: "list",
+      name: "backend",
+      message: "Which backend language would you like?",
+      choices: ["TypeScript (Node/Express)", "Python (Flask)"],
+    });
+  }
+
+  if (!options.api) {
+    prompts.push({
+      type: "list",
+      name: "api",
+      message: "Which API would you like?",
+      choices: ["REST", "GraphQL"],
+    });
+  }
+
+  if (!options.database) {
+    prompts.push({
+      type: "list",
+      name: "database",
+      message: "Which database would you like?",
+      choices: ["PostgreSQL", "MongoDB"],
+    });
+  }
+
+  if (!options.auth) {
+    prompts.push({
+      type: "confirm",
+      name: "auth",
+      message: "Would you like built-in auth features?",
+      default: false,
+    });
+  }
+
+  const answers = await inquirer.prompt(prompts);
+
+  return {
+    ...options,
+    backend: options.backend || answers.backend,
+    api: options.api || answers.api,
+    database: options.database || answers.database,
+    auth: options.auth || answers.auth,
+  };
+};
+
+const confirmPrompt = async (options: Options) => {
+  const message =
+    `You have chosen to create a ${options.backend} app with a ` +
+    `${options.api} API, ${options.database} database, and ${
+      options.auth ? "" : "no "
+    }built-in auth. Please confirm:`;
+
+  const prompt = {
+    type: "confirm",
+    name: "confirm",
+    message,
+    default: false,
+  };
+  const answers = await inquirer.prompt([prompt]);
+  return answers.confirm;
+};
+
+const cli = async (args: CommandLineArgs) => {
+  /* eslint-disable-next-line no-console */
+  console.log(
+    boxen(
+      chalk.bold(
+        figlet.textSync("create-bp-app", { horizontalLayout: "full" }),
+      ),
+      {
+        padding: 1,
+        margin: 1,
+        borderStyle: "double",
+        borderColor: "blue",
+      },
+    ),
+  );
+  let options = parseArguments(args);
+  options = await promptOptions(options);
+  const confirm = await confirmPrompt(options);
+  if (confirm) {
+    /* eslint-disable-next-line no-console */
```

Comment:
Since we're actually allowing outputting to the console, I'm fine with just allowing this in the project so you can move to this to the eslint config.